src/utils/api/transaction/lsk.js

```diff
@@ -1,11 +1,144 @@
-export const getTransaction = data => new Promise(resolve =>
-  resolve({ endpoint: 'getTransaction', token: 'LSK', data }));
+import http from '../http';
+import ws from '../ws';
+import transactionTypes from '../../../constants/transactionTypes';
+import { getDelegates } from '../delegate';
 
-export const getTransactions = data => new Promise(resolve =>
-  resolve({ endpoint: 'getTransactions', token: 'LSK', data }));
+/**
+ * Retrieves the details of a single transaction
+ *
+ * @param {Object} data
+ * @param {String} data.params - Id of the transaction
+ * @param {String?} data.baseUrl - Lisk Service API url to override the
+ * existing ServiceUrl on the network param. We may use this to retrieve
+ * the details of an archived transaction.
+ * @param {Object} data.network - Network setting from Redux store
+ * @returns {Promise} Transaction details API call
+ */
+export const getTransaction = data => http({
+  path: 'transactions',
+  params: { id: data.id },
+  network: data.network,
+  baseUrl: data.baseUrl,
+});
 
-export const getRegisteredDelegates = data => new Promise(resolve =>
-  resolve({ endpoint: 'getRegisteredDelegates', token: 'LSK', data }));
+const txFilters = {
+  dateFrom: { key: 'from', test: str => typeof str === 'string' },
+  dateTo: { key: 'to', test: str => typeof str === 'string' },
+  amountFrom: { key: 'min', test: str => typeof str === 'string' },
+  amountTo: { key: 'max', test: str => typeof str === 'string' },
```

Comment:
According to the API documentation the amounts are in Beddows format. I would validate that the parameter is a number.